set.mm

```diff
@@ -409744,6 +409795,33 @@ is a bijection ( 1-1 onto function) in a simple graph.  (Contributed
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
+  $c ComplGraph $.
+  $c ComplUSGraph $.
+
+  $( Extend class notation with (arbitrary) complete graphs. $)
+  ccplgr $a class ComplGraph $.
+
+  $( Extend class notation with complete simple graphs. $)
+  ccusgr $a class ComplUSGraph $.
+
+  ${
+    $d g v $.
+    $( Define the class of all complete "graphs".  A class/graph is called
+       _complete_ if every pair of distinct vertices is connected by an edge,
+       i.e. each vertex has all other vertices as neighbors.  (Contributed by
+       AV, 24-Oct-2020.) $)
+    df-cplgr $a |- ComplGraph = { g | A. v e. ( Vtx ` g )
+                                       v e. ( UnivVtx ` g ) } $.
+  $}
+
+  $( Define the class of all complete simple graphs.  A simple graph is called
+     _complete_ if every pair of distinct vertices is connected by a (unique)
+     edge, see definition in section 1.1 of [Diestel] p. 3.  In contrast, the
+     definition in section I.1 of [Bollobas] p. 3 is based on the size of
+     (finite) complete graphs, see ~ cusgrsize .  (Contributed by Alexander van
+     der Vekens, 12-Oct-2017.)  (Revised by AV, 24-Oct-2020.) $)
+  df-cusgr $a |- ComplUSGraph = { g e. USGraph | g e. ComplGraph } $.
```

Comment:
possible to define it as an intersection ? ( USGraph i^i ComplGraph )  This saves a dummy variable

Reply:
Good idea, I will earmark it for later PR (add TODO-AV ;-) ).